### PR TITLE
Roloc fix totally broken

### PR DESCRIPTION
**Pull request by rabidmonkeyman**
fixing the roloc taking control of the UI at all times no matter what is or is not connected. also fixed it spamming the ever loving $&*$ out of the console with i2c errors if it wasnt connected

### CODE REVIEW

File: rolocController.cpp [hunk 1]
```diff
@@ -79,6 +81,16 @@ void ROLOCcontroller::init()
     // init dbus
     mDbusHandler.init();
 
+    //init the bad packet timeout
+    mpDisconnectTimer = new QTimer();
+    QObject::connect(mpDisconnectTimer, &QTimer::timeout, [=]()
```
Comment (dmacattack) [inline rolocController.cpp:86]: this capture is "by-copy" in this case I usually do "this" when accessing member variables. im kinda surprised itll allow the copy capture for mDusHandler

File: rolocController.cpp [hunk 2]
```diff
@@ -242,7 +233,7 @@ void ROLOCcontroller::processRolocData()
                 double stdDev = qSqrt(getVariance(mDepthAccumulator));
 #if DBG_BLOCK
                 qDebug() << "*************************** bit set  -> " <<  CHECK_BIT(rolocData, 1)  << "*********************";
-                qDebug() << "smaple size: " << mNumSamples;
+                qDebug() << "sample size: " << mNumSamples;
```
Comment (dmacattack) [inline rolocController.cpp:236]: smaple ... good ctach

**Review by dmacattack — APPROVED** [review]

File: rolocController.cpp (hunk 1 above)
Comment (rabidmonkeyman) [reply rolocController.cpp]: youre right, ill change it

File: rolocController.cpp (hunk 2 above)
Comment (rabidmonkeyman) [reply rolocController.cpp:236]: it happened to catch my eye while i was scrolling

**Pull request merged**